src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -256,14 +256,20 @@ protected Object readResolve() {
         return this;
     }
 
-    public Instance instance() {
+    public Instance instance() throws IOException {
```

Comment:
A corresponding catch block for this exception type should be added to the call site in the provision method.